docs/products/flink/howto/pg-cdc-connector.rst

```diff
@@ -0,0 +1,83 @@
+Create a PostgreSQL® CDC connector-based Apache Flink®
+===========================================================
+
+Change Data Capture (CDC) is a technique that enables the tracking and capturing of changes made to data within a PostgreSQL® database. By identifying and capturing changes at the granular row level, CDC enables applications to react and promptly process these changes in real time. This ensures up-to-date data accuracy and minimizes the latency involved in processing updates.
+
+When using Aiven for Apache Flink®, you can seamlessly leverage the power of the PostgreSQL CDC Connector to stream and process real-time data changes from PostgreSQL databases. Integrated with the Debezium engine, the CDC connector captures changes at the granular level of each table within an event stream. 
+
+This article provides you with information on how to create a PostgreSQL CDC connector-based Apache Flink® table. 
+
+Prerequisites
+--------------
+Before setting up the PostgreSQL CDC Connector with Aiven for Apache Flink, ensure that you have the following prerequisites in place:
+
+* Running Aiven for Apache Flink® Service
+* Running Aiven for Apache Kafka® Service
+* Running Aiven for PostgreSQL® Service
+* Integration between Aiven for Flink and Aiven for PostgreSQL: Establish the necessary integration between your Aiven for Apache Flink service and Aiven for PostgreSQL service. 
+
+In addition to the above, gather the following information about the source PostgreSQL database:
+
+* ``Hostname``: The hostname or IP address of the PostgreSQL server where your source database is located.
+* ``Port``: The port number on which the PostgreSQL server is listening for incoming connections.
+* ``Database name``: The name of the source database within the PostgreSQL server.
+* ``Username``: The username used to authenticate and access the PostgreSQL database.
+* ``Password``: The password associated with the provided username for authentication.
+* ``Schema name``: The schema name where the source table is located within the database.
+* ``Table name``: The name of the source table from which you want to capture data changes.
+* ``Decoding plugin name``: The decoding plugin name to use for capturing the changes. For PostgreSQL CDC, set it as ``pgoutput``.
+
+.. important:: 
+    To create a PostgreSQL CDC source connector in Aiven for Apache Flink with Aiven for PostgreSQL using the pgoutput plugin, you need to have superuser privileges.
```

Comment:
Maybe add here a link how to enable it from our kafka connector blog?

Reply:
Included a troubleshooting section and cross-linked to this note.